client/app/src/main/java/com/microsoft/research/karya/ui/registration/CreationCodeFragment.kt

```diff
@@ -98,6 +118,7 @@ class CreationCodeActivity : BaseActivity(useAssistant = true) {
      * Else set the error message appropriately.
      */
     private fun verifyCreationCode(creationCode: String) {
+
         ioScope.launch {
             val callForCreationCodeCheck = karyaAPI.checkCreationCode(creationCode)
             if (callForCreationCodeCheck.isSuccessful) {
```

Comment:
Don't call the api from the fragment. Create a stub method instead and remove `ioScope.launch`

Reply:
You mean it to call from repo?

Reply:
Yeah but just create an empty method for it for now.

Reply:
Could we add a TODO instead? We would then not break the functionality

Reply:
Sure, a TODO works for now. But this means we'll have to revisit it again. A stub would not break the application, it'll just not perform any tasks.

Reply:
Sure, a TODO works for now. But this means we'll have to revisit it again. A stub would not break the application, it'll just not perform any tasks.

Reply:
Yeah sorry :P , by no longer runs I meant the particular functionality no longer runs as the method is empty. I will add the TODO